cedar-drt/fuzz/fuzz_targets/abac-type-directed.rs

```diff
@@ -28,22 +28,27 @@ use cedar_policy_generators::{
 };
 use libfuzzer_sys::arbitrary::{self, Arbitrary, Unstructured};
 use log::{debug, info};
+use serde::Serialize;
 use std::convert::TryFrom;
 
 /// Input expected by this fuzz target:
 /// An ABAC hierarchy, policy, and 8 associated requests
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize)]
 struct FuzzTargetInput {
     /// generated schema
+    #[serde(skip)]
     pub schema: Schema,
     /// generated entity slice
+    #[serde(skip)]
     pub entities: Entities,
     /// Should we pre-evaluate entity attributes
+    #[serde(skip)]
     pub should_cache_entities: bool,
     /// generated policy
     pub policy: ABACPolicy,
     /// the requests to try for this hierarchy and policy. We try 8 requests per
     /// policy/hierarchy
+    #[serde(skip)]
     pub requests: [ABACRequest; 8],
 }
```

Comment:
I would expect `serialize` to include most of these fields.  Can we just call `serialize` on `.policy` if this is what we want?

Reply:
Nah, because I need it to be the FuzzTargetInput so I can log all of them via the macro easily.